README.md

```diff
@@ -85,6 +85,30 @@ $trustpilot->merchant()->findReview((string) $reviewId);
 $trustpilot->merchant()->getReviews((array) $serviceReviewsFilterOptions);
 ```
 
+## Caching
```

Comment:
I would add this part under the Development section. Is there anything that we should mention on Caching on the Usage section?

Reply:
@AndreiaMatiasSilva I have updated some of the parts of the code and included usage example.